backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Service/UserService.java

```diff
@@ -49,5 +49,16 @@ public User saveUser(User user) {
         user.setPassword(Security.hashPassword(user.getPassword()));
         return userRepository.save(user);
     }
+
+    public boolean updateUser(Long id, User user) {
+        if (userRepository.existsById(id)) {
+            User existingUser = userRepository.findById(id).get();
+            existingUser.setPassword(user.getPassword());
```

Comment:
You have to hash the password, please use the 
`Security.hashPassword(user.getPassword())`